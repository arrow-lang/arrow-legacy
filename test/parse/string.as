// A simple string literal.
"Hello World";

// Some escape sequences.
"\\";
"\'";
"\"";
"\a";
"\b";
"\f";
"\n";
"\r";
"\t";
"\v";
// "\x7e";
// "\xc2\xa7";
