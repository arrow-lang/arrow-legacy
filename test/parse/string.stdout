{
  "tag": "Module",
  "span": "test/parse/string.as:1.1-2.13",
  "name": "string",
  "block": {
    "tag": "Block",
    "span": "test/parse/string.as:1.1-2.13",
    "statements": [
      {
        "tag": "String",
        "span": "test/parse/string.as:1.1-14",
        "text": "Hello World"
      },
      {
        "tag": "String",
        "span": "test/parse/string.as:2.1-13",
        "text": "Goodbye \\n"
      }
    ]
  }
}
